### ISSUE TITLE
riscv64: Python/perf_jit_trampoline.c:491:6: error: #error "Unsupported target architecture"

### ISSUE THREAD

**Issue by furkanonder** [python/cpython#121138]
### Bug description:

The problem is seen in a [buildbot](https://buildbot.python.org/all/#/builders/1438/builds/129).

```
$ uname -a
Linux DietPi 6.1.81 #1 SMP Sat Mar  9 21:40:38 UTC 2024 riscv64 GNU/Linux
```

### Configuration
```sh
./configure --prefix '$(PWD)/target'
```
Build Output:
```sh
Python/perf_jit_trampoline.c:441:9: note: in definition of macro ‘DWRF_SECTION’
  441 |         stmt;                                                                                           \
      |         ^~~~
Python/perf_jit_trampoline.c:458:18: note: in expansion of macro ‘DWRF_U8’
  458 |                  DWRF_U8(DWRF_REG_RA); /* Return address register. */
      |                  ^~~~~~~
Python/perf_jit_trampoline.c:458:26: note: each undeclared identifier is reported only once for each function it appears in
  458 |                  DWRF_U8(DWRF_REG_RA); /* Return address register. */
      |                          ^~~~~~~~~~~
Python/perf_jit_trampoline.c:441:9: note: in definition of macro ‘DWRF_SECTION’
  441 |         stmt;                                                                                           \
      |         ^~~~
Python/perf_jit_trampoline.c:458:18: note: in expansion of macro ‘DWRF_U8’
  458 |                  DWRF_U8(DWRF_REG_RA); /* Return address register. */
      |                  ^~~~~~~
Python/perf_jit_trampoline.c:461:53: error: ‘DWRF_REG_SP’ undeclared (first use in this function)
  461 |                  DWRF_U8(DWRF_CFA_def_cfa); DWRF_UV(DWRF_REG_SP); DWRF_UV(sizeof(uintptr_t));
      |                                                     ^~~~~~~~~~~
Python/perf_jit_trampoline.c:441:9: note: in definition of macro ‘DWRF_SECTION’
  441 |         stmt;                                                                                           \
      |         ^~~~
Python/perf_jit_trampoline.c:461:45: note: in expansion of macro ‘DWRF_UV’
  461 |                  DWRF_U8(DWRF_CFA_def_cfa); DWRF_UV(DWRF_REG_SP); DWRF_UV(sizeof(uintptr_t));
      |                                             ^~~~~~~
Python/perf_jit_trampoline.c:430:64: warning: left-hand operand of comma expression has no effect [-Wunused-value]
  430 | #define DWRF_UV(x) (ctx->p = p, elfctx_append_uleb128(ctx, (x)), p = ctx->p)
      |                                                                ^
Python/perf_jit_trampoline.c:441:9: note: in definition of macro ‘DWRF_SECTION’
  441 |         stmt;                                                                                           \
      |         ^~~~
Python/perf_jit_trampoline.c:461:45: note: in expansion of macro ‘DWRF_UV’
  461 |                  DWRF_U8(DWRF_CFA_def_cfa); DWRF_UV(DWRF_REG_SP); DWRF_UV(sizeof(uintptr_t));
      |                                             ^~~~~~~
Python/perf_jit_trampoline.c:491:6: error: #error "Unsupported target architecture"
  491 | #    error "Unsupported target architecture"
      |      ^~~~~
make: *** [Makefile:3033: Python/perf_jit_trampoline.o] Error 1
make: *** Waiting for unfinished jobs....
program finished with exit code 2
elapsedTime=271.078532
```

### CPython versions tested on:

3.13

### Operating systems tested on:

Linux

**Comment by brandtbucher** [comment]
@sobolevn: Let's only use `topic-JIT` label for issues that reproduce with `--enable-experimental-jit` (but *not* `--enable-experimental-jit=interpreter`).

This particular issue is about the `perf` trampoline, which happens to be a JIT, but not the one the label is intended for. :)

**Comment by furkanonder** [comment]
duplicate of https://github.com/python/cpython/issues/121201
